Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,129 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## DataSource manipulation
+
+By call this method whenever the datasource changes, to notify the datasource may have. Need to notify the datasource action(add, remove and reset) through notify listener. In the following example, the appointment is added and notifyListeners is called in onPressed callback of the FloatingActionButton.
+
+#### Add Action
+
+When dynamically adding appointments to a data source, we must notify the data source.
```

Comment:
we must notify the data source for the add action

Reply:
I have checked and changed into we must notify the data source for the add action based on the action. I have updated my changes in latest commit.